docs/1.getting-started/10.data-fetching.md

```diff
@@ -194,10 +194,10 @@ The `useAsyncData` composable is a great way to wrap and wait for multiple `$fet
 
 ```vue
 <script setup lang="ts">
-const { data: discounts, status } = await useAsyncData('cart-discount', async () => {
+const { data: discounts, status } = await useAsyncData('cart-discount', async (_, { signal }) => {
```

Comment:
this might be helpful for people to know what the argument is:

```suggestion
const { data: discounts, status } = await useAsyncData('cart-discount', async (_nuxtApp, { signal }) => {
```